scripts/run-e2e-android.sh

```
@@ -1,15 +1,12 @@
 #!/bin/bash
+set -uo pipefail
 set -x
 
-cd example || exit
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+source "$SCRIPT_DIR/metro.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./metro.sh was not specified as input (see shellcheck -x).

(SC1091)